### PR TITLE
Ignore docs changes on Azure Pipelines and CirrusCI

### PR DESCRIPTION
**Pull request by joerick**
Saves a bit of pointless CI time. Jenkins and Circle don't support it.

### CODE REVIEW

**Comment by Czaki** [comment]
Maybe this https://github.com/bjd2385/dynamic-continuation-orb

File: .cirrus.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+skip: changesIncludeOnly('docs/*')
```
Comment (henryiii) [inline .cirrus.yml:1]: What about `.pre-commit-config.yaml`?

**Review by henryiii — APPROVED** [review]

File: .cirrus.yml (hunk 1 above)
Comment (joerick) [reply .cirrus.yml:1]: good idea!

**Comment by joerick** [comment]
> Maybe this https://github.com/bjd2385/dynamic-continuation-orb

I didn't look at this, but there was also a first-party 'orb' that did something similar. But it seemed way more complicated than what I was looking for, I think it's more for multi-project monorepo setups.

If you can figure it out, please do!

**Pull request merged**